src/screens/Feed.js

```diff
@@ -30,7 +30,7 @@ export default class FeedScreen extends React.Component {
     const isLiked = likers[uid] === true ? true : false; // if i liked it
     const isBookmarked = bookmarkers[uid] === true ? true : false;
     return (
-      <TouchableWithoutFeedback onPress={() => { }}>
+      <TouchableWithoutFeedback onPress={() => {}}>
```

Comment:
We should add a console.log() statement here to make sure the TouchableWithoutFeedback works as intended

Reply:
Removed <TouchableWithoutFeedback onPress={() => {}}>